### PR TITLE
Delete or rename shell history: .ash_history

### LINKED ISSUE #1605: Delete or rename shell history: .ash_history
**Describe the bug**

The "Delete or rename shell history" rule detects changes to bash, zsh, and fish history files, but does not detect changes to `.ash_history`. The ash_history file is common for alpine containers, so it might be a good one to include.

https://github.com/falcosecurity/falco/blob/master/rules/falco_rules.yaml#L2612-L2623

**How to reproduce it**

Delete or truncate the .ash_history file in an alpine container.

**Expected behaviour**

Falco would alert for the "Delete or rename shell history" rule.

**Environment**

- Falco version: 0.27.0

- System info:

- Cloud provider or hardware configuration:
- OS: Alpine container running on Amazon Linux nodes

- Kernel:

- Installation method: Falco running as a daemonset in Kubernetes

**Additional context**

### PR DESCRIPTION
**Pull request by bdashrad**
**What type of PR is this?**

/kind rule-update

**Any specific area of the project related to this PR?**

/area rules

**What this PR does / why we need it**:

Includes .ash_history in `rule: Delete or rename shell history`

**Which issue(s) this PR fixes**:

Fixes #1605

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
rule(macro truncate_shell_history): include .ash_history
rule(macro modify_shell_history): include .ash_history
```

### CODE REVIEW

**Comment by leogr** [comment]
Closing and reopening to trigger the CI
/close

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -2575,22 +2575,26 @@
       evt.arg.name endswith "zsh_history" or
       evt.arg.name contains "fish_read_history" or
       evt.arg.name endswith "fish_history" or
+      evt.arg.name contains "ash_history" or
       evt.arg.oldpath contains "bash_history" or
       evt.arg.oldpath endswith "zsh_history" or
       evt.arg.oldpath contains "fish_read_history" or
       evt.arg.oldpath endswith "fish_history" or
+      evt.arg.oldpath contains "ash_history" or
```
Comment (darryk10) [inline rules/falco_rules.yaml:2583]: Would be possible to use endswith operator instead of contains?
Thanks

Comment (leogr) [reply rules/falco_rules.yaml:2583]: Good idea @darryk10 :+1:

Comment (Kaizhe) [reply rules/falco_rules.yaml:2583]: If we introduce `ash_history` (doesn't matter whether using operator `contains` or `endswith`), we can ignore the `bash_history` then. A proper comment will be useful here saying both `bash_history` and `ash_history` will be covered.

Comment (bdashrad) [reply rules/falco_rules.yaml:2583]: Seems like a reasonable approach to me. Do we still want to use `contains` or do we want to change it to `endswith`. I don't have a preference.

**Comment by leogr** [comment]
@Kaizhe ping :hugs:

**Review by Kaizhe — APPROVED** [review]

**Pull request merged**